package.json

```diff
@@ -22,7 +22,7 @@
         "update-deps": "npm update",
         "postupdate-deps": "bower update",
         "prestart": "npm install",
-        "start": "http-server -a localhost -p 3000 -c-1 ./app",
+        "start": "http-server -a localhost -p 4000 -c-1 ./app",
```

Comment:
Will check if port 4000 is going to support on my PC or not

Reply:
ok